variables.tf

```diff
@@ -44,6 +44,25 @@ variable "additional_public_subnets" {
   }))
 }
 
+variable "bastion_host_ami_name_filter" {
+  description = "The AMI filter to use for the bastion host's AMI."
+  type        = string
+  default     = "amzn2-ami-hvm-2.*-x86_64-ebs"
+
+}
+
+variable "bastion_host_ami_owners" {
+  description = "The list of owners used to select the AMI."
+  type        = list(string)
+  default     = ["amazon"]
+}
+
+variable "bastion_host_ami_id" {
+  type        = string
+  description = "The ID of the AIM to use for the instance. Setting this will ignore `bastion_host_ami_name_filter` and `bastion_host_ami_owners`."
```

Comment:
```suggestion
  description = "The ID of the AMI to use for the instance. Setting this will ignore `bastion_host_ami_name_filter` and `bastion_host_ami_owners`."
```